webknossos/webknossos/cli/convert.py

```diff
@@ -37,14 +40,21 @@ def main(
         ),
     ],
     voxel_size: Annotated[
-        VoxelSize,
+        VoxelSizeTuple,
         typer.Option(
             help="The size of one voxel in source data in nanometers. "
             "Should be a comma separated string (e.g. 11.0,11.0,20.0).",
             parser=parse_voxel_size,
-            metavar="VOXEL_SIZE",
+            metavar="VoxelSize",
+            show_default=False,
         ),
     ],
+    length_unit: Annotated[
```

Comment:
this is the name of the cli argument, right? Maybe just unit or voxel_size_unit would be better here. While I think LenghtUnit is the right name for the type, the variable here can have more semantics in context

Reply:
Sounds good, then I would prefer unit, as it is shorter.